### PR TITLE
Update using-static.md

### PR DESCRIPTION
**Pull request by ghost**
Code update + link.

### CODE REVIEW

File: docs/csharp/language-reference/keywords/using-static.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ ms.author: "ronpet"
...
-The `using static` directive designates a type whose static members you can access without specifying a type name. Its syntax is:
+The `using static` directive designates a type whose static members and nested types you can access without specifying a type name. Its syntax is:
 
 ```csharp
-using static <fully-qualified-type-name>
+using static <fully-qualified-type-name>;
 ```
 
-where *fully-qualified-type-name* is the name of the type whose static members can be referenced without specifying a type name. If you do not provide a fully qualified type name (the full namespace name along with the type name), C# generates compiler error CS0246: "The type or namespace name '<type-name>' could not be found."
+where *fully-qualified-type-name* is the name of the type whose static members and nested types can be referenced without specifying a type name. If you do not provide a fully qualified type name (the full namespace name along with the type name), C# generates compiler error [CS0246](../compiler-messages/cs0246.md).
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/using-static.md]: I like the addition of the link, but the sentence "The type of namespace … could not be found." should not be deleted.

We don't want to repeat all the information in another article, but the one sentence means that readers don't need to click the link for the most basic information.

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for updating this @johnnyagerard 

The addition of referencing nested types is important. 

I did have one requested change before we merge this one.  

Thanks again.

**Comment by johnnyagerard** [comment]
@BillWagner I updated the compiler error message in the same time.

**Comment by rpetrusha** [comment]
Build seems stuck. Closing and reopening.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks for making these updates.  I'll :shipit: now.

**Pull request merged**